ckanext/unhcr/actions.py

```diff
@@ -1492,13 +1491,13 @@ def geography_autocomplete(context, data_dict):
     geos = sorted(geos, key=lambda x: x['similarity'], reverse=True)
     c = 1
     for geo in geos:
+        inactive_tag = '' if geo['gis_status'] == 'active' else 'INACTIVE '
+        geo['name'] = "{}{} [{}]".format(inactive_tag, geo['name'], geo['custom_layer_name'])
```

Comment:
This looks a bit better:

```suggestion
        geo['name'] = "{} [{}]".format(geo['name'], geo['custom_layer_name'])
        if geo['gis_status'] == 'active':
            geo['name'] = '*' + geo['name'] + ' - INACTIVE'
        
```